### PR TITLE
Update chart for terraform-k8s v0.2.1-beta

### PR DESCRIPTION
**Pull request by koikonom**

### CODE REVIEW

**Review by redeux — COMMENTED** [review]
Looks like Chart.yaml needs a version bump

https://github.com/hashicorp/terraform-helm/blob/18777ff2bfc88dbad30dc3d266bb10bedc3086f2/Chart.yaml#L3

**Comment by aareet** [comment]
A note to update the versions here once https://github.com/hashicorp/terraform-k8s/pull/86 is merged

File: Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: terraform
-version: 0.1.6-alpha
+version: 0.2.0-beta
```
Comment (redeux) [inline Chart.yaml:3]: Needs the version bump here too.

**Review by aareet — APPROVED** [review]

**Review by aareet — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.2.1 beta (Dec 14, 2020)
```
[the remark this answers is not in the document]
Comment (koikonom) [reply CHANGELOG.md:1]: oops :)

**Pull request merged**